soc/nordic/nrf53/Kconfig

```diff
@@ -208,7 +208,7 @@ config SOC_NRF53_CPUNET_MGMT
 
 config SOC_NRF53_CPUNET_ENABLE
 	bool "NRF53 Network MCU is enabled at boot time"
-	default y if NRF_802154_SER_HOST
+	default y if NRF_802154_SER_HOST || BT
```

Comment:
I would assume this change is not needed? We previously didn't need to set `SOC_NRF53_CPUNET_ENABLE` for Bluetooth applications

Reply:
If we are observing that 54H needs to set this for BT applications, but 53 for some reason does not, then
I would also support having 54H copy whatever 53 is doing at the moment.

As long as there is no unnecessary divergence between 53 and 54 I am happy :)